lib/routes/profile.js

```diff
@@ -65,6 +70,11 @@ module.exports = {
         if (creds.scope.indexOf('openid') !== -1) {
           result.sub = creds.user;
         }
+
+        if (result.avatar) {
+          result.avatarDefault = result.avatar === DEFAULT_AVATAR_URL;
```

Comment:
this helps keep the batch requests clean (currently they extract a single property and is too much to refactor, but we can in the future)

Reply:
Sounds good; might be worth leaving that comment in the code for our future selves as well.